az-pipelines/Dockerfile

```diff
@@ -6,8 +6,8 @@ RUN apk update \
 LABEL version="1.0.0"
 
 LABEL maintainer="Microsoft Corporation" 
-LABEL com.github.actions.name="GitHub Action to queue Azure pipeline" 
-LABEL com.github.actions.description="Queue Azure devops pipeline" 
+LABEL com.github.actions.name="Queue Azure devops pipeline" 
```

Comment:
Spec question: Should this not be Queue Azure devops 'Build' pipeline ? It is not intuitive whether its build or release pipeline.

Reply:
I have discussed this with Pulkit. He said We will have one pipeline for YAML which is currently Build and will evolve to Single pipeline & other for “classic” release.